src/components/Modals/RenameProjectModal.js

```diff
@@ -36,8 +39,8 @@ export const RenameProjectModal = () => {
   const [renameProjectMutation] = useMutation(RENAME_PROJECT_MUTATION);
 
   const renameProject = () => {
-    const newName = document.getElementById('name').value
-    renameProjectMutation({variables: {id: project.id, name: newName}, onCompleted: onCompleted})
+    console.log('renaming')
```

Comment:
`console.log` here and on line 32

sorry for the comment spam, if you're answer to any of these is 'yes I want `console.logs`' ignore my comments